### ISSUE TITLE
Google signing is not working

### ISSUE THREAD

**Issue by williambout** [williambout/paper-for-mac#1]

**Comment by plainspace** [comment]
Awesome project! What does this issue mean exactly? Do you mean Google Auth requiring the modal to open?

**Comment by williambout** [comment]
@plainspace Yes, right now it opens a browser window, but instead the app should open a window.

**Comment by plainspace** [comment]
Ah. What does it take to fix this?

**Comment by ahowarda** [comment]
I think this is what you are trying to do. [https://developers.google.com/identity/sign-in/web/sign-in?authuser=2](url) Follow the link procedures, would set the ID up for Paper, but I'm not sure if you want it through your Google API Manager @williambout  I think this is what will help the fix.

**Comment by williambout** [comment]
@ahowarda Thanks for the link, I'll take a look.

**Comment by zeroby0** [comment]
@williambout Isn't opening in browser window actually helpful? People tend to save passwords in browser and sign in is easier.

**Comment by Natebeta** [comment]
So is there an fix now available? Would be great..

**Comment by gantoja** [comment]
Is there a fix available? I can't sign in using Google
